doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -5,269 +5,297 @@ Reasoning with inductive types
+     .. note::
+        To destruct a number, use the syntax :n:`destruct (@natural)` (not
+        very interesting anyway).
 
-.. tacn:: case @term
-   :name: case
+   :n:`@ident` (in :n:`@destruction_arg`)
+     If :token:`ident` is a quantified variable in the conclusion
+     of the goal, then :n:`destruct @ident` behaves like :tacn:`intros`
+     :n:`until @ident; destruct @ident`.
+     If :token:`ident` is a hypothesis of the context and :token:`ident`
+     is no longer dependent in the goal after application
+     of :tacn:`destruct`, it is erased (to avoid erasure, use parentheses, as
+     in :n:`destruct (@ident)`).
+
+   :n:`using @one_term {? with @bindings }`
+     Makes the tactic equivalent to :tacn:`induction` :n:`@induction_clause_list`.
+
+   :n:`@as_or_and_ipat`
+      Provides names for the variables introduced
+      into the context.  The :token:`as_or_and_ipat` must have the
+      form :n:`[ {*| {* @intropattern } } ]` with one :n:`{* @intropattern }`
+      for each constructor in the order in which the constructors are defined.
+      The items in
+      each :n:`{* @intropattern }` are used to name the variables created in
+      each new goal (one per constructor).  If there are not enough names,
+      Coq picks fresh names.
+
+   :n:`eqn : @naming_intropattern`
+      Defines new items in the context for the matching terms using the specified
+      or generated names.
+
+   :n:`with @bindings`  (in :n:`@one_term_with_bindings`)
+      Provides explicit instances for
+      the :term:`dependent premises <dependent premise>` of the type of
+      :token:`one_term`.
+
+   :n:`@occurrences`
+     Selects specific subterms of the goal and/or hypotheses to apply
+     the tactic to.  See :ref:`Occurrence clauses <occurrenceclauses>`.
+     If it occurs at the end of the :n:`@induction_clause_list`, then
+     there can only be one :n:`induction_clause`, which can't have its
+     own :n:`occurrences` clause.
+
+   .. tacn:: edestruct @induction_clause_list
+
+      Equivalent to :n:`destruct @term` except that it does not
+      fail if the instance of a dependent premises of the type
+      of :token:`term` is not inferrable. Instead, the unresolved instances
+      are left as existential variables to be inferred later, in the same way
+      they are in :tacn:`eapply`.
 
-   The tactic :n:`case` is a more basic tactic to perform case analysis without
-   recursion. It behaves as :n:`elim @term` but using a case-analysis
-   elimination principle and not a recursive one.
+.. tacn:: case @induction_clause_list
 
-.. tacv:: case @term with @bindings
+   .. todo PR: clear as mud? (below) the syntax accepts more parameters than elim
+      not sure what to say here
 
-   Analogous to :n:`elim @term with @bindings` above.
+   An older, more basic tactic to perform case analysis without
+   recursion. It behaves like :tacn:`elim` but uses a case-analysis
+   elimination principle rather than a recursive one.
```

Comment:
I suggest dropping the sentence with the reference to `elim` completely. However, I would add one to say that we recommend `destruct` when possible.